_data/sites.json

```diff
@@ -7082,6 +7123,16 @@
             "teachoo.com"
         ]
     },
+    
+    {
+        "name": "Tebex",
+        "url": "https://server.tebex.io/profile/delete",
+        "difficulty": "easy",
+        "notes": "Just click the 'delete profile' button in your account menu",
+        "domains": [
+            "tebex.io"
+        ]
+    },
```

Comment:
Tebex also needs to come after Teamviewer 😄

Reply:
Everything should be ok now, sorry again for this